react/src/components/SessionDetailContent.tsx

```diff
@@ -347,8 +351,44 @@ const SessionDetailContent: React.FC<{
               />
             </Descriptions.Item>
           ) : null}
-          <Descriptions.Item label={'Resource Usage'} span={md ? 2 : 1}>
-            <SessionUsageMonitor sessionFrgmt={session} />
+          <Descriptions.Item
+            label={
+              <Flex direction="column" align="start" gap={token.marginSM}>
+                <Typography.Text style={{ color: token.colorTextSecondary }}>
+                  {t('session.ResourceUsage')}
+                </Typography.Text>
+                <Select
+                  size="small"
+                  popupMatchSelectWidth={false}
+                  style={{ width: '100%' }}
```

Comment:
```suggestion
                  style={{ width: '100%', display: 'none' }}
```

Let's hide this selection until the problem where the percentage seems strange in the core is solved.